### DISCUSSION TITLE
Need help in using Biome with Neovim

### DISCUSSION THREAD

**Discussion by subhoghoshX** [biomejs/biome#1260] [Q&A] [answered]
I was really happy to stumble upon Biome as a linter/formatter with builtin LSP support in nvim-lspconfig couple months back. With the recent advancements in its development, I'm more excited/optimistic about it and want to try it in a project. I'll share the feedback on how it goes along the way and how it can be improved.

First, I'm having a little issue setting it up with Neovim. I'm following this -> https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#biome.

I'm focusing on the formatter part at the moment, I was expecting a command (like `:BiomeFormat` or something) or it to work with `lua vim.lsp.buf.format()`, but it doesn't seem to work.

I'm not too well versed in the LSP stuff, so please excuse me if I'm missing something obvious. Happy to provide more info/logs if needed. Thanks!

**Comment by ematipico** [comment, marked as answer]
It's possible that you use a version of the LSP client that doesn't support dynamic registration. We Dynamically register the formatting using dynamic capabilities. Dynamic capabilities may or may not be supported by the client.

This has changed though, and Biome's next release will register the formatting at startup, without the need of dynamic registration.

**Reply by subhoghoshX** [reply]
I installed the nightly version i.e. `@biomejs/biome@1.4.1-nightly.e087146`, and it seem to work now. Thanks a lot for the help :)
